src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,33 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        LocalDate localDateFrom = LocalDate.parse(dateFrom, FORMATTER);
+        LocalDate localDateTo = LocalDate.parse(dateTo, FORMATTER);
+        StringBuilder result = new StringBuilder().append("Report for period ")
+                .append(dateFrom).append(" - ").append(dateTo);
+        String[] arrayOfData;
+
+        for (String name : names) {
+            int salary = 0;
+            for (String line : data) {
+                arrayOfData = line.split(" ");
+                LocalDate currentDate = LocalDate.parse(arrayOfData[0], FORMATTER);
+                if (localDateFrom.compareTo(currentDate) <= 0
+                        && localDateTo.compareTo(currentDate) >= 0
```

Comment:
use more appropriate methods of LocalDate for this condition

Reply:
Can`t find more appropriated method, need prompt & help for this. I check isAfter, isBefore, isEqual but they dont work.

Reply:
You are right, you need to use isAfter, isBefore, isEqual (the should work)